SCClassLibrary/JITLib/ProxySpace/NodeProxy.sc

```diff
@@ -83,11 +84,11 @@ NodeProxy : BusPlug {
 	}
 
 	fadeTime_ { | dur |
-		if(dur.isNil) { this.unset(\fadeTime) } { this.set(\fadeTime, dur) };
+		this.set(\fadeTime, dur ? defaultFadeTime) // Use default if fadeTime set to nil 
```

Comment:
no need for a comment, the code speaks for itself!

Reply:
Thanks for the feedback. I added it on @snappizz s request https://github.com/supercollider/supercollider/pull/4462#discussion_r306614776

Reply:
So I can remove or keep it, either way is fine by me to be honest but let me know what you guys want :)

Reply:
hehe :) For me, the sentence `"this.set(\fadeTime, dur ? defaultFadeTime)"` reads almost like English, and doesn't differ from the comment. I tend to reserve comments for places where the code doesn't do what it seems to be doing.  @snappizz is it OK without a comment now that we have the `defaultFadeTime` variable?

Reply:
see https://github.com/supercollider/supercollider#discussion_r306614776 -- the comment is more to emphasize the intended semantics (which do carry a small amount of surprise) than explain the behavior. i agree a comment helps with that.

Reply:
Ok, I see - it is because someone might suspect that it is just a band-aid protection rather than intended (the link just goes to the main page, what should it point to?).

Reply:
should link to a discussion comment by nathan -- either an issue with GitHub or your browser.

Reply:
I've tried with different browsers, all behave the same.

Reply:
i am not GitHub's support team :)

Reply:
The link doesn't work for me either.
if you explain how you generated it I volunteer to file a bug with github (?)

Reply:
>i am not GitHub's support team :)
I mentioned it so that you know that I am not able to follow your argument. But let's leave it as it is.